### ISSUE TITLE
[Bug]: Mac version nil pointer when joining network

### ISSUE THREAD

**Issue by mlschuh** [gravitl/netmaker#1428]
### Contact Details

mitch.schuh@gmail.com

### What happened?

Using the GUI for netclient, when joining a network, the netclient software threw a nil pointer error after `starting wireguard`.  After this, `netclient list` was reporting that it was joined and the peer list was updating, however the underlying wireguard connection was not established. 

### Version

v0.14.6

### What OS are you using?

Mac

### Relevant log output

```shell
% sudo netclient
[netclient] 2022-07-27 16:42:40 joining netmaker at XXXXX
[netclient] 2022-07-27 16:42:41 UDP hole punching enabled for node XXXXXXX
[netclient] 2022-07-27 16:42:41 starting wireguard 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x43b8607]

goroutine 9 [running]:
github.com/gravitl/netmaker/netclient/wireguard.deleteInterface({0xc00044e7e4?, 0xb?}, {0xc0004062d0?, 0x6?})
	/Users/runner/work/netmaker/netmaker/netclient/wireguard/mac.go:154 +0xe7
github.com/gravitl/netmaker/netclient/wireguard.RemoveConfMac({0xc00044e7e4, 0xb})
	/Users/runner/work/netmaker/netmaker/netclient/wireguard/mac.go:32 +0x3f
github.com/gravitl/netmaker/netclient/wireguard.WgQuickDownMac(0xc0001376d8, {0xc00044e7e4?, 0xc000136cb0?})
	/Users/runner/work/netmaker/netmaker/netclient/wireguard/mac.go:18 +0x2d
github.com/gravitl/netmaker/netclient/wireguard.ApplyMacOSConf(0xc00021f840?, {0xc00044e7e4, 0xb}, {0xc0001c4c30, 0x2f})
	/Users/runner/work/netmaker/netmaker/netclient/wireguard/unix.go:36 +0x32
github.com/gravitl/netmaker/netclient/wireguard.ApplyConf(0xc0001376d8, {0xc00044e7e4?, 0x5258700?}, {0xc0001c4c30?, 0x98b629b3f?})
	/Users/runner/work/netmaker/netmaker/netclient/wireguard/common.go:295 +0x2c
github.com/gravitl/netmaker/netclient/wireguard.InitWireguard(0xc0001376d8, {0xc0001c4240?, 0x1?}, {0xc00029ac00?, 0x3, 0x4}, 0x0)
	/Users/runner/work/netmaker/netmaker/netclient/wireguard/common.go:180 +0x5ad
github.com/gravitl/netmaker/netclient/functions.JoinNetwork(0xc000271800, {0x0, 0x0})
	/Users/runner/work/netmaker/netmaker/netclient/functions/join.go:192 +0x1085
github.com/gravitl/netmaker/netclient/gui/components/views.GetJoinView.func1()
	/Users/runner/work/netmaker/netmaker/netclient/gui/components/views/join.go:35 +0x177
fyne.io/fyne/v2/widget.(*Button).Tapped(0xc000365d60, 0xc000000201)
	/Users/runner/go/pkg/mod/fyne.io/fyne/v2@v2.2.2/widget/button.go:194 +0x53
fyne.io/fyne/v2/internal/driver/glfw.(*window).mouseClickedHandleTapDoubleTap.func1()
	/Users/runner/go/pkg/mod/fyne.io/fyne/v2@v2.2.2/internal/driver/glfw/window.go:634 +0x26
fyne.io/fyne/v2/internal/driver/common.(*Window).RunEventQueue(0x0?)
	/Users/runner/go/pkg/mod/fyne.io/fyne/v2@v2.2.2/internal/driver/common/window.go:35 +0x3e
created by fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).createWindow.func1
	/Users/runner/go/pkg/mod/fyne.io/fyne/v2@v2.2.2/internal/driver/glfw/window.go:942 +0x136
```

### Contributing guidelines

- [X] Yes, I did.

**Comment by afeiszli** [comment]
hotfix is merged in https://github.com/gravitl/netmaker/pull/1429

We will have updated Mac binaries/installer later tonight or tomorrow morning

**Comment by mattkasun** [comment]
Brew updated

**Comment by afeiszli** [comment]
fixed via hotfix #1429
